web/ui/dashboard-component/projects/convoy-dashboard/src/lib/convoy-dashboard.component.scss

```diff
@@ -40,3 +40,14 @@
 		}
 	}
 }
+
+.underline {
+	text-decoration: underline var(--primary-color);
+}
+
+input[type='search']::-webkit-search-decoration,
+input[type='search']::-webkit-search-cancel-button,
+input[type='search']::-webkit-search-results-button,
+input[type='search']::-webkit-search-results-decoration {
+	display: none;
+}
```

Comment:
can we move this to the ui library

Reply:
Yea sure, I'll make a PR there. It won't just make a new ui-library release because of this. So the update will go out in the next ui-library release. @Oluwadaminiola

Reply:
okay make sense